play/src/front/Api/Iframe/Ui/UIWebsite.ts

```diff
@@ -351,9 +351,9 @@ export class UIWebsiteCommands extends IframeApiContribution<UIWebsiteCommands>
      * Returns one UI website (iframe positionned on the viewport) by ID.
      * {@link https://workadventu.re/map-building/api-ui.md#get-all-ui-websites | Website documentation}
      * @param {string} id The id of the UIWebsite
-     * @returns {Promise<UIWebsite>} Promise to return UI website
+     * @returns {Promise<UIWebsite | undefined>} Promise to return UI website
      */
-    async getById(id: string): Promise<UIWebsite> {
+    async getById(id: string): Promise<UIWebsite | undefined> {
```

Comment:
It was due to the compiler complaining about possible undefined down the line. Theoretically you can pass any string as the key thus it might indeed return an undefined value.